extensions/interactions/FractionInput/directives/fraction_input_interaction_directive.html

```diff
@@ -9,7 +9,7 @@
     <[getWarningText()]>
   </div>
 </form>
-</div>
```

Comment:
Why remove this?

Reply:
Have you noticed, it was redundant? There is not corresponding `<div>` tag for this.

Reply:
No there is corresponding `<div>` tag, I just checked.

Reply:
There is `<div>` tag pair at line no- 8 and 10 while this one is redundant.